redis-persistence/src/main/java/com/netflix/conductor/redis/dao/RedisExecutionDAO.java

```diff
@@ -726,6 +727,31 @@ public void removeEventExecution(EventExecution eventExecution) {
         }
     }
 
+    @Override
+    public boolean hasAccess(Object[] args, List<String> labels) {
+        return false;
+    }
+
+    @Override
+    public boolean exists(Object[] args) {
+        return false;
+    }
+
+    @Override
+    public List<String> getUserWorkflowIds(List<String> labels) {
+        return List.of();
+    }
+
+    @Override
+    public List<String> getPresentIds(List<String> ids) {
+        return List.of();
+    }
+
+    @Override
+    public SearchResult<String> getSearchResultIds(List<String> rolesAndGroupsList) {
+        return null;
```

Comment:
throw new UnsupportedOperationException("Not implemented yet.");

Reply:
Resolved.